go.mod

```diff
@@ -46,14 +48,14 @@ require (
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.55.0
 	// Prometheus maps version 2.x.y to tags v0.x.y.
-	github.com/prometheus/prometheus v0.53.1-0.20240625160125-1abeebacb870
+	github.com/prometheus/prometheus v0.53.2-0.20240718123124-e9dec5fc537b
 	github.com/segmentio/fasthash v1.0.3
 	github.com/sony/gobreaker v1.0.0
 	github.com/spf13/afero v1.11.0
 	github.com/stretchr/testify v1.9.0
 	github.com/thanos-io/objstore v0.0.0-20240622095743-1afe5d4bc3cd
-	github.com/thanos-io/promql-engine v0.0.0-20240515161521-93aa311933cf
-	github.com/thanos-io/thanos v0.36.0-rc.0
+	github.com/thanos-io/promql-engine v0.0.0-20240718195911-cdbd6dfed36b
+	github.com/thanos-io/thanos v0.35.2-0.20240722172812-990a60b72647
```

Comment:
Seems like we downgraded from thanos version `v0.36.0-rc.0` to `v0.35.2-0`?

Reply:
![990a60b72647](https://github.com/thanos-io/thanos/commit/990a60b726477848504c2e286a38771fb58239c0) is newer. 


![v0.36.0-rc.0](https://github.com/thanos-io/thanos/tree/v0.36.0-rc.0) is from one month ago.